Resources/public/css/views/dashboard/allcontent-dashboard-block.css

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+.ez-view-dashboardblockallcontentview .pure-table {
+    width: 100%;
+    border: 0;
+    table-layout: fixed;
+}
+
+.ez-view-dashboardblockallcontentview .ez-allcontent-block-head {
+    box-sizing: border-box;
+    padding: .5em;
+}
+
+.ez-view-dashboardblockallcontentview .ez-allcontent-block-wrapper {
+    box-sizing: border-box;
+    border: 1px solid;
+    position: relative;
+}
+
+.ez-view-dashboardblockallcontentview.is-loading .ez-allcontent-block-content {
+    height: 3em;
+    display: block;
+}
+
+.ez-view-dashboardblockallcontentview .ez-allcontent-block-header th:nth-child(1),
+.ez-view-dashboardblockallcontentview .ez-allcontent-block-content td:nth-child(1) { width: 30%; }
+.ez-view-dashboardblockallcontentview .ez-allcontent-block-header th:nth-child(2),
+.ez-view-dashboardblockallcontentview .ez-allcontent-block-content td:nth-child(2) { width: 30%; }
+.ez-view-dashboardblockallcontentview .ez-allcontent-block-header th:nth-child(3),
+.ez-view-dashboardblockallcontentview .ez-allcontent-block-content td:nth-child(3) { width: 10%; }
+.ez-view-dashboardblockallcontentview .ez-allcontent-block-header th:nth-child(4),
+.ez-view-dashboardblockallcontentview .ez-allcontent-block-content td:nth-child(4) { width: 30%; }
+
+
+.ez-view-dashboardblockallcontentview .ez-allcontent-block-cell {
+    position: relative;
+    z-index: 1;
+    box-sizing: border-box;
+}
+
+.ez-view-dashboardblockallcontentview .ez-allcontent-block-row-options {
+    transform: translate3d(100%, -50%, 0) scaleX(0);
```

Comment:
not exactly, that's right for the translate3d but not for the scaleX part which actually means hide the element. But you hide it in a way that makes possible to animate the display of the options. This is typically something that should go in the theme stylesheet.
Instead, here you should have a `display: none` (or `visibility: hidden`) and in the theme stylesheet, you should them write a `display: flex` (or `visibility: visible`) + the transform + the transition.